sunbeam/commands/init.py

```diff
@@ -106,6 +107,12 @@ def init(auto: bool, role: str) -> None:
     model = 'sunbeam'
     bundle: Path = snap.paths.common / 'etc' / 'bundles' / 'control-plane.yaml'
 
+    states_path: Path = snap.paths.common / 'etc' / 'bundles' / 'states.json'
+    with open(states_path) as states_data:
+        states = json.load(states_data)
+
+    timeout = 900
```

Comment:
I suspect this should probably be configurable. We can change it in a future PR, but it probably doesn't hurt to have this be a config option within the snap.

Reply:
Note: Configurable in the next commit.